src/api/json/catalog.json

```
@@ -4232,11 +4232,47 @@
     },
     {
       "name": "StrmPrivacy batch job configuration file",
-      "description": "StrmPrivacy derived from the BatchJob protobuf definition",
+      "description": "StrmPrivacy batch job (https://docs.strmprivacy.io/docs/latest/concepts/data-processing/batch-jobs/)",
       "fileMatch": ["*batch-job-config*.json", "*batch-job-config*.yaml"],
-      "url": "https://docs.strmprivacy.io/jsonschema/BatchJob.json",
+      "url": "https://json-schema.api.strmprivacy.io/latest/strmprivacy.api.entities.v1.BatchJob.json",
       "versions": {
-        "1.0": "https://docs.strmprivacy.io/jsonschema/BatchJob.json"
+        "1.0": "https://json-schema.api.strmprivacy.io/latest/strmprivacy.api.entities.v1.BatchJob.json"
+      }
+    },
+    {
+      "name": "StrmPrivacy SimpleSchema",
+      "description": "StrmPrivacy SimpleSchema (https://docs.strmprivacy.io/docs/latest/quickstart/data-contracts/simple-schema/)",
```

```suggestion
      "description": "StrmPrivacy SimpleSchema\nhttps://docs.strmprivacy.io/docs/latest/quickstart/data-contracts/simple-schema/",
```

The same issue as above.